<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="0" hostname="DESKTOP-GA60R37" failures="0" tests="5" name="Default test" time="0.738" errors="0" timestamp="2023-10-03T22:20:05 IST">
  <testcase classname="testcases.LoginTest" name="@BeforeTest setup" time="0.618">
    <failure type="java.lang.NullPointerException" message="null value in entry: url=null">
      <![CDATA[java.lang.NullPointerException: null value in entry: url=null
at com.google.common.collect.CollectPreconditions.checkEntryNotNull(CollectPreconditions.java:33)
at com.google.common.collect.SingletonImmutableBiMap.<init>(SingletonImmutableBiMap.java:43)
at com.google.common.collect.ImmutableBiMap.of(ImmutableBiMap.java:81)
at com.google.common.collect.ImmutableMap.of(ImmutableMap.java:128)
at org.openqa.selenium.remote.DriverCommand.GET(DriverCommand.java:187)
at org.openqa.selenium.remote.RemoteWebDriver.get(RemoteWebDriver.java:297)
at methods.Base.launchApp(Base.java:44)
at testcases.LoginTest.setup(LoginTest.java:34)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:77)
at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.base/java.lang.reflect.Method.invoke(Method.java:568)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:133)
at org.testng.internal.MethodInvocationHelper.invokeMethodConsideringTimeout(MethodInvocationHelper.java:62)
at org.testng.internal.ConfigInvoker.invokeConfigurationMethod(ConfigInvoker.java:385)
at org.testng.internal.ConfigInvoker.invokeConfigurations(ConfigInvoker.java:321)
at org.testng.TestRunner.invokeTestConfigurations(TestRunner.java:637)
at org.testng.TestRunner.beforeRun(TestRunner.java:627)
at org.testng.TestRunner.run(TestRunner.java:589)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:377)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:371)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:332)
at org.testng.SuiteRunner.run(SuiteRunner.java:276)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:53)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:96)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1212)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1134)
at org.testng.TestNG.runSuites(TestNG.java:1063)
at org.testng.TestNG.run(TestNG.java:1031)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
    </failure>
  </testcase> <!-- @BeforeTest setup -->
  <testcase classname="testcases.LoginTest" name="loginWithValidUsernameAndValidPassword" time="0.0">
    <skipped/>
  </testcase> <!-- loginWithValidUsernameAndValidPassword -->
  <testcase classname="testcases.LoginTest" name="loginWithValidUsernameAndInvalidPassword" time="0.0">
    <skipped/>
  </testcase> <!-- loginWithValidUsernameAndInvalidPassword -->
  <testcase classname="testcases.LoginTest" name="loginWithInvalidUsernameAndValidPassword" time="0.0">
    <skipped/>
  </testcase> <!-- loginWithInvalidUsernameAndValidPassword -->
  <testcase classname="testcases.LoginTest" name="loginWithInvalidUsernameAndInvalidPassword" time="0.0">
    <skipped/>
  </testcase> <!-- loginWithInvalidUsernameAndInvalidPassword -->
  <testcase classname="testcases.LoginTest" name="loginWithBlankUsernameAndInvalidPassword" time="0.0">
    <skipped/>
  </testcase> <!-- loginWithBlankUsernameAndInvalidPassword -->
</testsuite> <!-- Default test -->
